app/test/runTests.ts

```diff
@@ -89,13 +89,20 @@ async function runTest() {
   restrainedLog("│ " + chalk.cyan("Module") + ": " + chalk.yellow(testClassNames[classIndex]));
   restrainedLog("│ " + chalk.cyan("Running test") + ": " + chalk.magenta(currentlyRunningTest.name));
   restrainedLog("│");
-  let promise = currentlyRunningTest.apply(null, [db]);
-  promise.then(async () => {
-    nextTest();
-    if (!testsOver) {
-      await runTest();
-    }
-  });
+  await currentlyRunningTest.apply(null, [db]);
+  nextTest();
+  if (!testsOver) {
+    await runTest();
+  }
+  // let promise = currentlyRunningTest.apply(null, [db]);
```

Comment:
seems like we don't need this anymore

Reply:
yeah, I'll remove that real quick. There was really no reason to do it in a promise chain for testing, as we're awaiting inside our tests anyway.

Reply:
Ok cool, @Enochen should be gone now.

Reply:
wait, gotta pull into dev first lmao

Reply:
ok @Enochen now we're good